src/Console/Command/CompileCommand.php

```diff
@@ -0,0 +1,205 @@
+            throw new \InvalidArgumentException('Target architecture must be one of "x86_64" or "aarch64"');
+        }
+
+        $io = new SymfonyStyle($input, $output);
+        $io->section('Compiling PHP and your Castor app PHAR archive into a standalone binary');
+
+        $spcBinaryPath = PlatformUtil::getCacheDirectory() . '/castor-php-static-compiler/spc';
+        $spcBinaryDir = \dirname($spcBinaryPath);
+
+        $this->setupSPC(
+            $spcBinaryDir,
+            $spcBinaryPath,
+            $io,
+            $os,
+            $arch,
+            $output
+        );
+
+        if (!$this->fs->exists($spcBinaryDir . '/buildroot/bin/micro.sfx') || $input->getOption('php-rebuild')) {
+            $this->installPHPBuildTools($spcBinaryPath, $spcBinaryDir, $io);
+
+            $phpExtensions = $input->getOption('php-extensions');
+
+            $this->downloadPHPSourceDeps(
+                $spcBinaryPath,
+                $phpExtensions,
+                $input->getOption('php-version'),
+                $spcBinaryDir,
+                $io
+            );
+
+            $this->buildPHP(
+                $spcBinaryPath,
+                $phpExtensions,
+                $arch,
+                $spcBinaryDir,
+                $io
+            );
+        }
+
+        $this->mergePHPandPHARIntoSingleExecutable(
+            $spcBinaryPath,
+            $input->getArgument('phar-path'),
+            $input->getOption('output'),
+            $spcBinaryDir,
+            $io
+        );
+
+        return Command::SUCCESS;
+    }
+
+    private function downloadSPC(string $spcSourceUrl, string $spcBinaryDestination, OutputInterface $output): void
+    {
+        $response = $this->httpClient->request('GET', $spcSourceUrl);
+        $contentLength = $response->getHeaders()['content-length'][0] ?? 0;
+
+        $outputStream = fopen($spcBinaryDestination, 'w');
+        $progressBar = new ProgressBar($output, (int) $contentLength);
+
+        if (false === $outputStream) {
+            throw new \RuntimeException(sprintf('Failed to open file "%s" for writing', $spcBinaryDestination));
+        }
+
+        foreach ($this->httpClient->stream($response) as $chunk) {
+            fwrite($outputStream, $chunk->getContent());
+            $progressBar->advance(\strlen($chunk->getContent()));
+        }
+
+        fclose($outputStream);
+        chmod($spcBinaryDestination, 0o755);
+
+        $progressBar->finish();
+    }
+
+    private function installPHPBuildTools(string $spcBinaryPath, string $spcBinaryDir, SymfonyStyle $io): void
+    {
+        $installSPCDepsProcess = new Process(
+            command: [
+                $spcBinaryPath, 'doctor',
+                '--auto-fix',
+            ],
+            cwd: $spcBinaryDir,
+            timeout: 5 * 60
+        );
+        $io->text('Running command: ' . $installSPCDepsProcess->getCommandLine());
+        $installSPCDepsProcess->mustRun(fn ($type, $buffer) => print ($buffer));
```

Comment:
```suggestion
        $installSPCDepsProcess->mustRun(fn ($type, $buffer) => print($buffer));
```
This is strange php-cs-fixer did not fix it!

Reply:
I've copy-pasted this code from https://github.com/jolicode/castor/blob/main/src/Console/Command/RepackCommand.php#L109

But I think it shouldn't have parenthesis anyway, [`print` is not a function](https://www.php.net/manual/en/function.print.php#refsect1-function.print-notes)

Reply:
Good catch!